exasol/analytics/audit/audit.py

```diff
@@ -0,0 +1,51 @@
+from inspect import cleandoc
+from typing import List
+
+from exasol.analytics.audit.columns import BaseAuditColumns
+from exasol.analytics.schema import (
+    Column,
+    SchemaName,
+    TableLikeName,
+    TableLikeNameImpl,
+)
+
+
+class TableDescription:
+    """
+    This class describes an SQL table by its attributes
+    * table (instance of (TableLikeName)) defining the name and optionally the
+      database schema of the SQL table
+    * columns the names and types of the columns of the SQL table
+
+    The class also offers a property `render_create` which renders the
+    attributes into an SQL CREATE TABLE statement.
+    """
+
+    def __init__(self, table: TableLikeName, columns: List[Column]):
+        self.table = table
+        self.columns = {c.name.name: c for c in columns}
+
+    @property
+    def render_create(self):
+        columns = ",\n  ".join(c.for_create for c in self.columns.values())
```

Comment:
What is `for_create`? I can't find this property in the `ColumnName`.

Reply:
See here https://github.com/exasol/advanced-analytics-framework/blob/main/exasol/analytics/schema/column.py#L17